### PR TITLE
Spotinst - adding support for instance health check validation

### PR DESCRIPTION
**Pull request by jeffnoehren**
##### SUMMARY
Adding option to wait for instances to return healthy signal when creating or updating spotinst elastigroup

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
Spotinst Module

##### ADDITIONAL INFORMATION
If the variable `health_check_type` is set and `wait_for_instances` is True when running a Spotinst Playbook, then we will wait until the target number of instances all return a HEALTHY signal

```
ok: [localhost] => {
    "result": {
        "changed": true, 
        "failed": false, 
        "group_id": "sig-20520f40", 
        "instances": [], 
        "message": "Created group Successfully."
    }
}
```

### CODE REVIEW

**Comment by kmcgrath** [comment]
Hello @gundalow. What is the next step to get this PR merged?  Tests are passing but the bot does not seem to have moved past that point.

**Comment by felixfontein** [comment]
Passing tests are not enough, reviews are required (see [here](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md#for-pull-request-submitters)). @talzur is the module's author and seems to be the (only) module's current maintainer, so he should take a look.

(In any case, this PR also needs a [changelog fragment](https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#changelogs-how-to) before it can be merged.)

**Review by talzur — APPROVED** [review]

**Comment by talzur** [comment]
@felixfontein , @jeffnoehren , @kmcgrath 
Looks good guys.

**Comment by kmcgrath** [comment]
thank you @talzur!

@jeffnoehren lets get the change log updates in accordance with the comment from @felixfontein.

**Comment by jeffnoehren** [comment]
ok just added the changelog fragment! I am not sure if @talzur needs to approve again or not?

**Review by kmcgrath — APPROVED** [review]

**Comment by kmcgrath** [comment]
ready_for_review

**Comment by kmcgrath** [comment]
LGTM

**Comment by kmcgrath** [comment]
bot_status

**Comment by jeffnoehren** [comment]
bot_status

**Comment by jeffnoehren** [comment]
bot_status

**Comment by jeffnoehren** [comment]
ready_for_review

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Thanks all, merged into `devel` for release in Ansible 2.8.

**Pull request merged**
